test/locationscale.jl

```diff
@@ -14,15 +9,15 @@ function test_location_scale_normal(μ::Float64,σ::Float64,μD::Float64,σD::Fl
     @test maximum(d) == maximum(dref)
     @test extrema(d) == (minimum(d), maximum(d))
 
-    #### Conversions
+    #### Promotions and conversions
 
-    @test convert(LocationScale{Float16}, μ, σ, ρ) == LocationScale{Float16,typeof(ρ)}(Float16(μ), Float16(σ), ρ)
-    @test convert(LocationScale{Float16}, d) == LocationScale{Float16,typeof(ρ)}(Float16(μ), Float16(σ), ρ)
+    @test typeof(d.µ) === typeof(d.σ)
+    # @test convert(LocationScale{Float16}, μ, σ, ρ) == LocationScale{Float16,typeof(ρ)}(Float16(μ), Float16(σ), ρ)
```

Comment:
Outcommented tests?

Reply:
yes sorry I should remove those. This convert is not valid in the general sense, not all Real numbers can be cast into a `Float` in the general case